### ISSUE TITLE
Display debug content without login.

### ISSUE THREAD

**Issue by ahmed-sigmalux** [bueltge/debug-objects#51]
How to display the debug helper on public frontend?

I need to debug our login form but I'm unable to activate the debug helper when I'm not logged in.

**Comment by bueltge** [comment]
HI @ahmed-sigmalux 
currently check the plugin always the capabilities of logged in users, so that you can't debug this without a login. It is more effort to change this inside the plugin and was not on focus. I'm sorry.

```php
	public function __construct() {

		if ( ! current_user_can( '_debug_objects' ) ) {
			return;
		}
```

**Comment by bueltge** [comment]
Hello @ahmed-sigmalux 
I have change a lot classes and add a method to check the capability on each point for the current user. You can try this current version from github to test it.

You must add also a small plugin, a custom plugin for your installation that set the variable for the check of the rights always to `true`.
`add_action( 'debug_objects_view', true )`

That's all, now the check for rights, logged in is not necessary and you will see all debug informations, there is active from the settings form.

I hope this helps you?

**Comment by beeing** [comment]
This looks good, is this published?

**Comment by bueltge** [comment]
Hi @beeing 
not in the plugin of wordpress.org, but here on github is inside the master branch. It would be fine you test it also and give me feedback, so that I will published this.

**Comment by bueltge** [comment]
Now, it is published - version 2.5.0
